discovery-provider/nginx_conf/main.lua

```diff
@@ -0,0 +1,125 @@
+require "resty.core"
+
+local resty_random = require("resty.random")
+local resty_rsa = require("resty.rsa")
+local httpc = require("resty.http")
+
+math.randomseed(string.byte(resty_random.bytes(1)))
+
+local _M = {}
+
+local rate_limit = tonumber(os.getenv("audius_openresty_rate_limit") or "1000")
+
+local public_url = os.getenv("audius_openresty_public_url") or ""
+
+local redirect_to = {}
+for discovery_provider in string.gmatch(os.getenv("audius_openresty_redirect_to") or "", "[^,]+") do
+    table.insert(redirect_to, discovery_provider)
+end
+
+local accept_redirect_from = {}
+for discovery_provider in string.gmatch(os.getenv("audius_openresty_accept_redirect_from") or "", "[^,]+") do
+    accept_redirect_from[discovery_provider] = true
+end
+
+local rsa_public_key = os.getenv("audius_openresty_rsa_public_key") or ""
+local rsa_private_key = os.getenv("audius_openresty_rsa_private_key") or ""
+if (#rsa_public_key == 0 or #rsa_private_key == 0) then
```

Comment:
`#` in front of var in lua is [length operator](https://stackoverflow.com/questions/17974622/what-does-mean-in-lua)

Reply:
```lua
function empty_string (str)
    return #str == 0
end

if (empty_string(rsa_public_key) or empty_string(rsa_private_key)) then
```